src/pages/Home.jsx

```diff
@@ -0,0 +1,18 @@
+import Button from "../components/button/Button";
+
+import "./Home.scss";
+
+export default function Home() {
+  return (
+    <>
+      <div>

```

Comment:
If you're not using this `div` you can remove it, `fragment` up above is doing the wrapper of the elements.

Reply:
i used the div to put the background-color, i can do it in the fragment?

Reply:
You're using this div: https://github.com/MarcelloAragoni/mentor/pull/1/files#r843867095

so what you need do is to remove the `<>` fragment, because you already have a single node of its components.